persist-cli/src/main/java/io/ballerina/persist/cmd/Push.java

```diff
@@ -215,6 +225,11 @@ private void loadJdbcDriver() throws BalException {
             throw new BalException("Error in jdbc driver path : " + e.getMessage());
         }
     }
+    private void populatePlaceHolder(HashMap<String, String> persistConfigurations)
+            throws BalException {
+        SyntaxTreeGenerator.populateConfiguration(persistConfigurations, Paths.get(this.sourcePath,
+                                this.configPath).toAbsolutePath());
```

Comment:
Why is `populateConfiguration` added in SyntaxTreeGenerator. AFAIK this does not generate any syntax trees.

Reply:
This is added to keep the toml SyntaxTree related functions to a single file we use two types of syntax trees in our implementation

Reply:
Then naming should be `TomlSyntaxTreeUtils`

Reply:
Shall we update with the restructuring?

Reply:
Ack